src/v2/view-builder/internals/BaseForm.js

```diff
@@ -62,14 +62,14 @@ export default Form.extend({
     }
   },
 
-  addCallouts () {
-    const warningMsgs = this.options.appState.get('messages');
-    if (warningMsgs && warningMsgs.value.length) {
-      const messageCallout = createCallout({
-        content: warningMsgs.value[0].message,
-        type: 'warning',
+  showMessages () {
+    // render messages as text
+    const messagesObjs = this.options.appState.get('messages');
+    if (messagesObjs && messagesObjs.value.length) {
+      const content = messagesObjs.value.map((messagesObj) => {
+        return messagesObj.message;
       });
-      this.add(messageCallout, '.o-form-error-container');
+      this.add(`<span class="ion-messages-container">${content.join(' ')}</span>`, '.o-form-error-container');
```

Comment:
i would not include `ion` in class name, its implementation detail.

Reply:
but `messages-container` is just too general?

Reply:
`terminal-messages` ?

Reply:
`terminal` would imply it's for `TerminalView` but this is generic container for any messages.

Reply:
any other name but definitely don't include `ion`

Reply:
yeah, I was trying to avoid `terminal` in the classname and I also felt `messages-container` is too generic. For now I think I am going to leave it as is we can update it later if we come up with something better. 

TBH I don't actually like the name `messages` that the API is sending. I would have chosen a different name for it 🤷‍♂️ and chose a corresponding classname.